scripts/prepare_compliance_data.py

```diff
@@ -0,0 +1,155 @@
+"""
+A script that takes the compliance dataset (the released version
+of which is at https://github.com/ga4gh/compliance/tree/master/test-data)
+and turns it into a directory bundle of binary and JSON files suitable
+for use by the reference server.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import shutil
+import argparse
+import os
+import sys
+from os import path
+from glob import glob
+import pysam
+
+
+class ComplianceDataMunger(object):
+    def __init__(self, args):
+        self.inputDirectory = args.inputDirectory
+        self.outputDirectory = args.outputDirectory
+
+        # get all the reference files (they'll be the ones with .fa extension)
+        self.referenceFiles = map(path.basename,
+                                  glob(path.join(self.inputDirectory, "*.fa")))
+
+        self.refsetsDirectory = path.join(self.outputDirectory,
```

Comment:
As @dcolligan pointed out above, we tend to use ``os.path.join`` and ``glob.glob`` elsewhere.